db/fixtures/miq_product_features.yml

```diff
@@ -1444,6 +1444,90 @@
 
+# Cloud Resource Pools
+- :name: Cloud Resource Pools
+  :description: Access Everything under Cloud Resource Pools
+  :feature_type: node
+  :identifier: resource_pool_cloud
+  :children:
+  - :name: View
+    :description: Display Lists of Cloud Resource Pools
+    :feature_type: view
+    :identifier: resource_pool_cloud_view
+    :children:
+    - :name: List
+      :description: Display Lists of Cloud Resource Pools
+      :feature_type: view
+      :identifier: resource_pool_cloud_show_list
+    - :name: Show
+      :description: Display Individual Cloud Resource Pools
+      :feature_type: view
+      :identifier: resource_pool_cloud_show
+  - :name: Operate
+    :description: Perform Operations on Cloud Resource Pools
+    :feature_type: control
+    :identifier: resource_pool_cloud_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Tags of Cloud Resource Pools
+      :feature_type: control
+      :identifier: resource_pool_cloud_tag
+    - :name: Manage Policies
+      :description: Manage Policies of Cloud Resource Pools
+      :feature_type: control
+      :identifier: resource_pool_cloud_protect
+  - :name: Modify
+    :description: Modify Cloud Resource Pools
+    :feature_type: admin
+    :identifier: resource_pool_cloud_admin
+    :children:
+    - :name: Remove
+      :description: Remove Cloud Resource Pools
+      :feature_type: admin
+      :identifier: resource_pool_cloud_delete
+
+# Infrastructure Resource Pools
+- :name: Infrastructure Resource Pools
+  :description: Access Everything under Infrastructure Resource Pools
+  :feature_type: node
+  :identifier: resource_pool_infra
+  :children:
+  - :name: View
+    :description: Display Lists of Infrastructure Resource Pools
+    :feature_type: view
+    :identifier: resource_pool_infra_view
+    :children:
+    - :name: List
+      :description: Display Lists of Infrastructure Resource Pools
+      :feature_type: view
+      :identifier: resource_pool_infra_show_list
+    - :name: Show
+      :description: Display Individual Infrastructure Resource Pools
+      :feature_type: view
+      :identifier: resource_pool_infra_show
+  - :name: Operate
+    :description: Perform Operations on Infrastructure Resource Pools
+    :feature_type: control
+    :identifier: resource_pool_infra_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Tags of Infrastructure Resource Pools
+      :feature_type: control
+      :identifier: resource_pool_infra_tag
+    - :name: Manage Policies
+      :description: Manage Policies of Infrastructure Resource Pools
+      :feature_type: control
+      :identifier: resource_pool_infra_protect
+  - :name: Modify
+    :description: Modify Infrastructure Resource Pools
+    :feature_type: admin
+    :identifier: resource_pool_infra_admin
+    :children:
+    - :name: Remove
+      :description: Remove Infrastructure Resource Pools
+      :feature_type: admin
+      :identifier: resource_pool_infra_delete
+
```

Comment:
To solve the UI navigation issue caused by using the same `resource_pool` identifier for both `cloud and infrastructure menu_section` for resource_pools  in [default_menu.rb](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/presenters/menu/default_menu.rb#L97), I've created separate identifiers for both infrastructure and cloud resource pools, which resolves the UI navigation problem.

Reply:
@jeffibm is it expected that the same product feature would cause navigation issues in the UI?

Reply:
Hey @agrare , 

I think we can reuse them.

For example - 

I duplicated an entry in `manageiq`, the identifiers are the same here (`physical_chassis_show_list`)
<img width="1044" alt="image" src="https://github.com/ManageIQ/manageiq/assets/87487049/31c2c8f3-eb7f-4855-85db-4d7eb7d77f52">

and added the corresponding entry in `ui-classic` - 
<img width="1210" alt="image" src="https://github.com/ManageIQ/manageiq/assets/87487049/d131f612-ce19-489e-8546-c0cd616139f4">

Menu method definition
<img width="940" alt="image" src="https://github.com/ManageIQ/manageiq/assets/87487049/59719d47-87af-45c6-9c7f-39869f2243d5">


This worked and would redirect to the same URL when clicked on `Chassis` and `Chassis2`.
In this case `/physical_chassis/show_list#/`
<img width="529" alt="image" src="https://github.com/ManageIQ/manageiq/assets/87487049/defd7532-d031-4944-9460-cd5430e99035">

From the above example in ui-classic, I have used the menu item-id as `physical_chassis2`. 
The only problem I have seen so far is,  If we use `physical_chassis` instead, it is going to complain about duplicate ids.

<img width="897" alt="image" src="https://github.com/ManageIQ/manageiq/assets/87487049/c1c4199f-8d19-4b40-961c-9ade1abc5af5">

Reply:
> This worked and would redirect to the same URL when clicked on Chassis and Chassis2.

Hm this might be the problem @Guddetisandeep is having though, I don't think we want them redirecting to the same page.